exe/server/Dockerfile

```diff
@@ -1,17 +1,16 @@
-FROM golang:1.13.4 as builder
+FROM golang:1.13.7 as builder
+RUN apt-get update && apt-get install -y mesa-opencl-icd ocl-icd-opencl-dev
 RUN mkdir /app 
 WORKDIR /app 
-COPY go.mod go.sum ./
 COPY extern extern
+COPY go.mod go.sum ./
 RUN go mod download
 COPY . . 
-RUN GOOS=linux go build -o main exe/server/main.go
+RUN GOOS=linux go build -o main exe/server/main.go && \
+go run github.com/GeertJohan/go.rice/rice append --exec main -i ./build
```

Comment:
The _interesting_ part of embedding the `parameter.json` file in the executable.